### PR TITLE
Update size and add size-badge

### PR DESCRIPTION
**Pull request by StephanHoyer**
Maybe we should point to master?

### CODE REVIEW

**Comment by tivac** [comment]
Yes, that should point to `master`.

Did the build task that kept readme size values up to date stop working at some point?

**Comment by pygy** [comment]
@tivac maybe... I remembered it being automated so I didn't check.

**Comment by pygy** [comment]
Well, I briefly pointed it to `master`, but it looks like there's an extra build commit and I'm not sure where it comes from...

**Comment by tivac** [comment]
Sorry, I meant the badge should use `master`. This PR landing on `next` is correct.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![](http://img.badgesize.io/MithrilJS/mithril.js/next/mithril.min.js.svg?compression=gzip)
```
Comment (tivac) [inline README.md]: Badge should show the size on `master`, not `next`.

**Comment by tivac** [comment]
#1898 addressed this in a bigger rework. Thanks!

**Pull request closed without merging**